scopes/pkg/pkg/pkg.main.runtime.ts

```diff
@@ -206,7 +206,10 @@ export class PkgMain {
    * This is used in cases you want to actually run the components and make sure all the dependencies (especially peers) are resolved correctly
    */
   getRuntimeModulePath(component: Component, options: GetModulePathOptions = {}) {
-    const relativePath = this.dependencyResolver.getRuntimeModulePath(component);
+    const relativePath = this.dependencyResolver.getRuntimeModulePath(component, {
+      workspacePath: this.workspace.path,
+      rootComponentsPath: this.workspace.rootComponentsPath,
```

Comment:
don't you need the `isInWorkspace` prop here?

Reply:
I guess I don't need it as it wasn't here before my changes.